package-lock.json

```diff
@@ -0,0 +1,9213 @@
+{
```

Comment:
Try to avoid using both `npm` and `yarn`. They usually lead to a lot of problems in the future.

I believe we've used yarn with this project so let's stick to it.